### PR TITLE
Use global variable for CDN root directory

### PR DESCRIPTION
**Pull request by diarmidmackenzie**
**Description:**

As per #5119 

**Changes proposed:**

Replace hardcoded instances of `https://cdn.aframe.io/` with a global variable AFRAME_ROOT_CDN.

Based on @vincentfretin's suggestion [here](https://github.com/aframevr/aframe/issues/5119#issuecomment-1382817113)

### CODE REVIEW

File: src/constants/index.js [hunk 1]
```diff
@@ -2,5 +2,6 @@ module.exports = {
   AFRAME_INJECTED: 'aframe-injected',
   DEFAULT_CAMERA_HEIGHT: 1.6,
   DEFAULT_HANDEDNESS: 'right',
-  keyboardevent: require('./keyboardevent')
+  keyboardevent: require('./keyboardevent'),
+  AFRAME_CDN_ROOT: window.AFRAME_CDN_ROOT || 'https://cdn.aframe.io/'
```
Comment (vincentfretin) [inline src/constants/index.js:6]: This list seems to be sorted, you should put the new constant first, before AFRAME_INJECTED.

**Review by vincentfretin — COMMENTED** [review]
Thanks for doing this work.

File: src/constants/index.js (hunk 1 above)
Comment (diarmidmackenzie) [reply src/constants/index.js]: Yes, well spotted.  Now alphabetized.

**Comment by diarmidmackenzie** [comment]
> The text in https://github.com/aframevr/aframe/blob/master/docs/introduction/faq.md#can-i-use-a-frame-offline-or-self-hosted would also need to be modified.

I'd like to handle that as a separate PR.

I don't think the text as it stands *needs* to change for this PR to be OK - everything it says is still true, even after this code change.

More work needed to fully solve #5119, but I'd like to tackle this one bit at a time.

**Comment by nightgryphon** [comment]
it also be cool to add an assets archive for loacal deployment. Including fonts, controllers models etc. I has to manualy figure out which files are required for local deployment and download one by one...

**Comment by nightgryphon** [comment]
@vincentfretin Ohh! I've missed that repo. It can be a good idea to add a comment line with repo url near the constant and later in the docs. Also it is good to note which folders are actually required.

**Comment by dmarcos** [comment]
Thanks!

**Pull request merged**